### ISSUE TITLE
ExporterCore zipkin issue no trace logged

### ISSUE THREAD

**Issue by SebHonorine** [SteeltoeOSS/steeltoe#67]
Hello there,

I m trying to add distributed tracing to some projects (to create some demo for other teams). So I added some steeltoe into the mix.

I followed some stuff already present online like :

https://steeltoe.io/docs/steeltoe-management/#2-3-exporting

http://www.hlike.cn/article.php?id=15296

https://zipkin.io/

startup.cs
```
using Steeltoe.Management.Tracing;
using Steeltoe.Management.Exporter.Tracing; 

            services.AddDistributedTracing(Configuration);

            // Export traces to Zipkin
            services.AddZipkinExporter(Configuration);

            // Start up trace exporter
            app.UseTracingExporter();
```

my appSettings.json looks like

```
  "management": {
    "tracing": {
      "alwaysSample": true,
      "egressIgnorePattern": "/api/v2/spans|/v2/apps/.*/permissions|/eureka/.*|/oauth/.*",
      "exporter": {
        "zipkin": {
          "endpoint": "http://localhost:9411/api/v2/spans",
          "validateCertificates": false
        }
      }
    }
  },
```
My program.cs
```
                .ConfigureLogging((builderContext, loggingBuilder) =>
                {
                    loggingBuilder.ClearProviders();
                    loggingBuilder.AddConfiguration(builderContext.Configuration.GetSection("Logging"));
                    loggingBuilder.AddDynamicConsole();
                });
```
My log is set to true as exportable
info: Microsoft.AspNetCore.Hosting.Internal.WebHost[1]
       [AccesTi.Carts,f98fa5f63f5ed284,3842eb84bd4ec954,,true] Request starting HTTP/1.1 GET https://localhost:44300/api/Carts   
but I also see (don't know if its relevant but seems fishy) :

```
dbug: Steeltoe.Management.Tracing.Observer.HttpClientCoreObserver[0]
      HandleStartEvent: Ignoring path: /api/v2/spans
dbug: Steeltoe.Management.Tracing.Observer.HttpClientCoreObserver[0]
      HandleStopEvent: Missing span context
```

![image](https://user-images.githubusercontent.com/8523321/59713242-ec039080-91dc-11e9-9957-966d3329400a.png)

I don't know why my request are not sent to the zipkin server i must be missing something.

thanks for the help

**Comment by jkonicki** [comment]
Hi @SebHonorine,

This appears to be an issue with the Zipkin version.  Something broke somewhere between 2.5.0 and the latest 2.12. We are looking into the exact version, but haven't prioritized that yet.  We also opened a ticket with OpenCensus.  Once we investigate some more, we will let you know.  But for the time being 2.5.0 of Zipkin should work.  If you want to test out later version and identify the last know working version and report back, that would be helpful in identifying what might have changed.  

Thank you.

**Comment by hananiel** [comment]
Somewhere around version 2.12.2 of zipkin server,  there was a change. The open census team owns that part of the code and there is an issue that appears to have been resolved but not released: census-instrumentation/opencensus-csharp#136. One option is to run this version of zipkin server that works: `docker run -d -p 9411:9411 openzipkin/zipkin:2.12.2` or build open census-csharp from source or find their dev feed.

**Comment by SebHonorine** [comment]
i ll try this version of zipkin and keep you guys updated.

Thank you!

**Comment by SebHonorine** [comment]
YES thank you !!! it was indeed an issue with 2.12.9 I just started some test with 2.12.2 and it works fine. . . a day lost. . . but thanks.

**Comment by jkonicki** [comment]
Reopening since this needs to be fixed for the newer zipkin releases.  

1) We need to wait for the fix in OpenTelemetry (formerly OpenCensus project).  Note: this fix is in, we are just waiting on a release from that project.
2) There are some other areas of the Steeltoe code that need to be investigated and addressed. 

For now, use the noted version of zipkin 2.12.2.
